packages/types/src/index.ts

```diff
@@ -582,52 +585,6 @@ export type Station = {
   y: number;
 };
 
-export type ModeOption = {
```

Comment:
The breaking change was actually about the loss of these exports.

Reply:
I believe you can still import them but you have to specify the /deprecated file in your import.

Reply:
... which is a breaking change!

Reply:
I'm confused about this comment, there *is* a breaking change commit here for this reason

Reply:
What's the consensus here? Does removing these exports cause a breaking change?

Reply:
Yes it is a breaking change and it is fine.

Reply:
Are you implying that we should bring back these exports?

Reply:
@AdrianaCeric No, because there is already "BREAKING CHANGE" commit that announces this kind of stuff.